runelite-client/src/main/java/net/runelite/client/config/RuneLiteConfig.java

```diff
@@ -127,14 +127,16 @@ default boolean requestFocusOnNotification()
 	}
 
 	@ConfigItem(
-		keyName = "notificationSound",
-		name = "Enable sound on notifications",
+		keyName = "notificationSound_",
+		name = "Notification sound",
 		description = "Enables the playing of a beep sound when notifications are displayed",
 		position = 22
 	)
-	default boolean enableNotificationSound()
+	default Notifier.NativeCustomOff notificationSound()
 	{
-		return true;
+		return System.getProperty("os.name").toLowerCase().contains("win")
```

Comment:
This should be set to OFF by default imo.

Reply:
I set this to what most application default to: Toast+Sound.

I could add a OsSpecific<T> type that will marshal a map<OSName, T> in the config.

Reply:
Most applications default to using system notification. What puzzles me is why you chose custom notification on Windows as default. With os-specific config you are probably taking your PR way too far beyond is original intention, what was just add custom notifications, but now you are also adding custom sounds, changing RuneLite settings, all in one PR, I don't understand why. The custom sounds at least make sort of sense, so be it. But, why custom sound should be default for Windows, that do not makes sense. But anyway, as long as the os-specific configuration is not resolved, I would avoid it. Also, there is already code for detecting OS, why are you not using that one.

Reply:
The reason these are both on by default on windows is that the windows-native notifications are garbage. Their clickbox is roughly double size of the UI element. Clicking on them does not make them go away. Disabling them doesn't remove the clickbox. They force applications out of full screen mode. You cannot turn the volume down without editing the file. They do not work over remote desktop. They are __entirely broken__. Many applications (Chrome, Firefox, Anything using electron) that I use have no user-facing option to use native notifications on windows.

Reply:
Well, still better than having os-specific persisted configuration. And, the users have ability to choose. Also, what is the reasoning for custom sound default on windows instead of system beep, noone was reporting problems with that one.

Reply:
Because the system beeps (and notifications) are handled in a weird way, you cannot easily turn the volume down, so if you have most applications turned down, they will deafen you. They are two halves of the same problem, so I figured I should fix them at the same time.

Reply:
Also the system beep is not the notification sound, though they are the same by default on windows. Also people have complained about it. See #2275

Reply:
That do not sounds like problem to me, as people are used to system beep on Windows, and I would assume they have tweaked the volume of it accordingly. Also it respects your volume setting of your OS, so it is non-issue. And, that is not complain, that is feature request for having the sound customizable, like you did, but still, no reason to make it default behaviour. Like, I understand that you have this problem, but, there is option to fix it, so why not leave the system-specific settings out of this.

Reply:
Yeah the system beep is incredibly loud compared to every other audio source. I have to go into the volume mixer and reduce the volume for the app that sent the system beep to get it to a level that doesn't kill my ears

Reply:
Yeah that sounds like correct behaviour, I doubt any native OS notifications respect per-application sound, as notifications are not tied to applications, applications just send them to some handler. And yeah, well, you can change the config option, like everyone else after it will be added.

Reply:
That said, having different default configs based on OS is probably unnecessary, just providing the option is enough.

Reply:
I wouldn't mind defaulting it to off, if the option was visible in any way, but it is hidden.

Reply:
Maybe putting the `RuneLite` config section at the top of the menu would help with that.

Also, do we have any data on OS usage? If the majority of the players use windows then custom notifications should def be enabled by default

Reply:
With this I would result to by default not break anything for anyone. And custom notifications do mostly that for other OSes.